sunscreen_tfhe/gpu_src/src/math/fft/fft_f64_impl.cuh

```diff
@@ -23,19 +23,32 @@ SOFTWARE.
 #pragma once
 #include "../math.cuh"
 
+#if 1
 __device__ __inline__ double2 Get_W_value_f64(int N, int m)
 {
 	double2 ctemp;
-	sincos(-PI_2 * (double)m / (double)N, &ctemp.y, &ctemp.x);
+	sincos(-TAU * (double)m / (double)N, &ctemp.y, &ctemp.x);
 	return ctemp;
 }
 
 __device__ __inline__ double2 Get_W_value_inverse_f64(int N, int m)
 {
 	double2 ctemp;
-	sincos(PI_2 * (double)m / (double)N, &ctemp.y, &ctemp.x);
+	sincos(TAU * (double)m / (double)N, &ctemp.y, &ctemp.x);
 	return ctemp;
 }
+#else
+#include "fft_constants_f64.cuh"
```

Comment:
Our generated constant tables currently give the wrong result. In any case, recomputing is significantly faster (~40% with f32) than a memory operation, so I'd like to avoid  this if possible. However, the accuracy of the forward twisted FFT is 1e-9, which is pretty bad. I'll revisit this accuracy issue later, but I suspect it's cascading errors with sincos().